src/content/protocols/ccip.md

```diff
@@ -0,0 +1,384 @@
+
+Additionally, there is the _Committing Network_ which observes events on the source chain that signal messages that should be moved cross chain. If it detects such a transaction, the _Committing Network_ posts a root to the `CommitStore` contract on the destination chain.
+
+Lastly, there is the _Executing Network_ that observes the transactions and if roots are posted to the `CommitStore` and the roots are blessed, the executing network triggers the message to be executed on the target chain with the blessed messages.
+
+![Onchain Infrastructure](./diagrams/CCIP_Infrastructure.png)
+
+# Centralization
+
+## Governance
+
+The smart contracts of the CCIP protocol are governed by the `RBACTimelock` contract. The governance can change parameters and the onchain addresses of the nodes in each network (Risk, Committing, Executing).
+
+Governance actions are proposed by multichain multisigs (MCMSs) which have the Proposer role by calling `scheduleBatch` on the `RBACTimelock` contract (see [role owner table](#role-owners-rbactimelock)). The `RBACTimelock` contract enforces a minimum timelock, which is set to 3 hours. Particpants of governance can choose a specific timelock which is above or equal to the minimum timelock. The scheduled transactions are then ready to be executed after the timelock has passed. During the time of the exit window the scheduled transactions is object to cancellation by MCMSs with the Canceller role. With no cancellation during the exit window period execution becomes permissionless.
+
+A Bypasser role is assigned to a Multichain Multisig (MCMS) for the case of an emergency to execute transactions instantaneously without the possibility to cancel the transaction.
+
+The `RBACTimelock` (Governance) contract is deployed to any chain which CCIP is connecting.
+
+## Upgradeability
+
+This report did not analyse all smart contracts permissions that can be found in the smart contract foundation of the CCIP infrastructure. But three functions that expose upgradeability risk are `setConfig` on `RMN` contract, `setOCR2Config` on `CommitStore` and `setOCR2Config` on the `EVM2EVMOfframp` contract.
+
+These functions allow to set the signer and transmitter addresses of the cross-chain messages for the three different networks. Updating the representing addresses to a smaller set can introduce centralization risk that could be exploited to trigger malicious cross-chain messages, such as minting/releasing tokens on the destination chain, without sufficient collateralization on the origin chain.
+
+## Multichain Multisig (MCMS)
+
+Role owners of the `RBACTimelock` (governance) are multichain multisigs or MCMS for short. They allow to use the same signatures across many chains, as such only EOAs are supported and not smart contracts.
```

Comment:
Link the security council table. + 

I find the explanation a bit confusing. I would keep it to:

Multisigs are nodes. A signature is valid if all the children nodes reach their threshold.